SConstruct

```diff
@@ -871,6 +876,7 @@ if 'pulseaudio' in system_dependencies:
                 env.Die("can't find %s" % path)
 
             pulse_env.Append(LIBS=libs)
+            env['PKG_CONFIG_LIBS'] += libs
```

Comment:
1. The first line appends `libs` to `pulse_env`, but the second line appends `libs` to `env.` `pulse_env` is used to build PulseAudio modules, and these dependencies are specific to them. So the correct code would be:

```
            pulse_env.Append(LIBS=libs)
            pulse_env['PKG_CONFIG_LIBS'] += libs
```

2. Instead of manually duplicating libs to LIBS and PKG_CONFIG_LIBS every time, I suggest to add a method env.AddPkgConfigLibs(libs), which will append given `libs` both to `LIBS` and `PKG_CONFIG_LIBS` (using AppendUnique) and use it everywhere instead of `Append(LIBS=...)` and `AppendUnique(LIBS=...)`.